src/Stratis.Bitcoin.Features.Wallet/WalletManager.cs

```diff
@@ -158,8 +158,11 @@ public class WalletManager : IWalletManager
         public IWalletRepository WalletRepository { get; private set; }
 
         public uint256 WalletTipHash => this.WalletCommonTip(this.ChainIndexer.Tip)?.HashBlock ?? 0;
+
         public int WalletTipHeight => this.WalletCommonTip(this.ChainIndexer.Tip)?.Height ?? -1;
 
+        public bool IsStarted { get; private set; }
```

Comment:
We should look at having `State` property on all services in the future. But this is OK for now.